### ISSUE TITLE
Oil drilling rig

### ISSUE THREAD

**Issue by Quark-73** [PygmalionOfCyprus/cmo-db-requests#3986]
### DB Selector

Both

### Hypothetical

No

### Name

Oil Drilling Rig

### Facility Type

Supply

### Country

generic

### Service

Structure

### In Commission

1945 -

### Crew

approx. 200

### Sensors

usually none
but could be equipped with EW/ELINT/SIGINT - Equipment

### Mounts

Heli fuel capacity

### Magazines

none

### Aircraft & Docking Facilities

1 x Helipad
1 x small pier for service vessels (50 to 100 metres (160 to 330 ft) in length) 

### Comments

Should be only placed in water (if feasible not deeper than 500 m (1500 ft))
dimensions 300 x 400 x 300 wide x lenght x height
if possible: take damage over time by fire (similar to ships)

Oil rigs could be IMHO used as stepstone for helicopters in some scenarios. 

I made it supply type, since structure was not available in type drop down menu. 

### Sources

general information: https://en.wikipedia.org/wiki/Oil_platform

example dimensions: https://www.seattletimes.com/seattle-news/controversial-oil-drilling-rig-arrives-in-port-angeles/

**Comment by KLABDB3K** [comment]
Aren't these are in the DB just under "platforms".

**Comment by Quark-73** [comment]
I didn't found:
![platform](https://github.com/PygmalionOfCyprus/cmo-db-requests/assets/123901821/878f9098-251a-4179-a4d7-59a6acc598ad)

**Comment by Quark-73** [comment]
ah they are ships, my fault

**Comment by Quark-73** [comment]
I noticed offshore platforms have been added as facilities in 503. If I understand correctly, they are copies of the ship type platforms. It seems some data didn't came along with the copies. The facilities have no heli deck nor hangars like the ship ones. And this probably more an issue: they cannot placed in water.
Despite offshore platforms are more plausible as facility to me than as a ship, it would be okay for me when they stay just ships. I'm not sure if it's worth the effort to fully convert them.
